documentation/asciidoc/accessories/camera/camera_hardware.adoc

```diff
@@ -274,8 +274,11 @@ Other available schematics;
 This is the pinout of the 15-pin Camera Serial Interface (CSI) connector used on flagship Raspberry Pi models prior to the Raspberry Pi 5.
 The connector type is an Amphenol SFW15R-2STE1LF or a compatible equivalent.
 
-NOTE: Most official Camera Modules have a reversed pinout to accommodate 15-pin Type A / Type 1 FFC cables. This means Pin 1 on the Raspberry Pi connects to Pin 15 on the Camera Module, and vice versa.
-Direction is given from the perspective of the Raspberry Pi.
+NOTE: Most official Camera Modules use the same 15-pin connector but feature a reversed pinout to support 15-pin Type A / Type 1 FFC cables.
```

Comment:
"Most"?? I don't think there's an official camera module (see https://www.raspberrypi.com/products/#cameras-and-displays ) that _doesn't_ have a 15-pin connector?